ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/group_vars/CV_PATHFINDER_TESTS.yml

```diff
@@ -11,7 +11,6 @@ cv_pathfinder_regions:
     sites:
       - name: Site422
         id: 422
-        location: Somewhere
```

Comment:
Why is this removed?

Reply:
Good question

Reply:
[1ced490](https://github.com/aristanetworks/ansible-avd/pull/3446/commits/1ced490b155939fef4fc9e518fd958c28827894f)